datacube/drivers/postgres/sql.py

```diff
@@ -39,7 +39,12 @@ def visit_create_view(element, compiler, **kw):
 
 UPDATE_COLUMN_MIGRATE_SQL_TEMPLATE = """
 alter table {schema}.{table} add column if not exists updated
-timestamptz not null default now();
+timestamptz default null;
```

Comment:
Does this need to write rows? It's probably the default, defaulting to null anyway... Testing will show how it works :-)

Reply:
I was wondering the same thing. I assume it's the same as not mentioning a default... but we'll find out.